juicy-ace-editor.html

```diff
@@ -158,6 +160,12 @@
                 case "wrapmode":
                     this.editor.getSession().setUseWrapMode( newVal !== null );
                     break;
+                // non-Ace specific
+                case 'shadow-style':
+                    if(oldVal){
+                        this.shadowRoot.querySelector('#'+oldVal).remove();
```

Comment:
I think this `#` isn't needed, everywhere else it's documented as a CSS selector while this assumes an ID.